include/exadg/aero_acoustic/user_interface/field_functions.h

```diff
@@ -0,0 +1,53 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2024 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef EXADG_AERO_ACOUSTIC_USER_INTERFACE_FIELD_FUNCTIONS_H_
+#define EXADG_AERO_ACOUSTIC_USER_INTERFACE_FIELD_FUNCTIONS_H_
+
+#include <exadg/utilities/optional_function.h>
+
+namespace ExaDG
+{
+namespace AeroAcoustic
+{
+template<int dim>
+struct FieldFunctions
+{
+  /*
+   * The function source_term_blend_in is mainly used to blend in the source
+   * term over time. In some rare cases it might also be beneficial to fade
+   * out the source term in space: If large source term contributions are
```

Comment:
I did not fully understand "fade out in space": The `Function` would shrink the area in which the source term is active originally, or enlarge that area?

Reply:
It "shrinks" the area in the sense that one would apply a smooth function that is 1 in most of the domain and goes to 0 at boundaries.

Reply:
Could you add this explanation (if not already the case).

Reply:
[605ba8a](https://github.com/exadg/exadg/pull/644/commits/605ba8a104f745ab2a0b65d3aedcd93134b7ea8a)